source/blog/2016-10-04-ovirt-provider-ovn.md

```diff
@@ -0,0 +1,163 @@
+---
+title: oVirt Software Defined Networking, The OVN Network Provider
+author: mmirecki
+date: 2016-10-03 14:21:00 UTC
+tags: community, howto, blog, network
+comments: true
+published: true
+---
+
+oVirt offers not only its own internal networking, but also an API for external network providers. This API enables using external network management software inside environments managed by oVirt and takes advantage of their extended capabilities. 
+One of such solutions is OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that brings Software Defined Networking to [OVS](http://openvswitch.github.io/support/dist-docs/README.md.html).
+
+OVN enables support for virtual networks abstraction by adding native OVS support for virtual L2 and L3 overlays.
+This allows the user to create create as many VM networks as required, without troubling the adminstrator with vlan requests or infrastructure changes.
+
+READMORE
+
+The oVirt provider for OVN consists of two parts:
+* The oVirt OVN driver
+* The oVirt OVN provider
+
+## oVirt OVN Driver
+
+The oVirt OVN driver is the Virtual Interface Driver placed on oVirt hosts that handle the wiring of VM NICs to OVN networking.
+
+The driver allows Vdsm, libvirt, and OVN to interact whenever a NIC is plugged in such a way that the VM NIC is added to an appropriate OVN Logical Switch and the appropriate OVN overlays on all the hosts in the oVirt environment.
+
+The [oVirt OVN driver rpm](http://resources.ovirt.org/repos/mmirecki/ovirt-provider-ovn-driver-0-1.noarch.rpm) is now available for testing. The latest version can always be downloaded and built from the repository (described later in this article). Once the rpm is downloaded, it can be installed in the following way:
+
+    yum install ovirt-provider-ovn-driver-0-1.noarch.rpm
+
+OVN requires Vdsm and OVN (version 2.6 or later) to be installed on the host. The following OVN packages are required by the ovirt-provider-ovn-driver rpm:
+
+* `openvswitch`
+* `openvswitch-ovn-common`
+* `openvswitch-ovn-host`
+* `python-openvswitch`
+
+These are available from the [OVS website](http://openvswitch.org/download/) or built using the code downloaded from the OVS repo (described below).
+
+The OVN-controller service can be started using the following command:
+
+    systemctl start ovn-controller
+
+After installing the driver and OVN, the OVN-controller must be configured. This can be done either using the vdsm-tool:
+
+    vdsm-tool ovn-config <OVN central IP> <local IP used for OVN tunneling>
```

Comment:
To clarify the parameters requirement, I would change "OVN central IP" to "OVN central server IP", and add that the second parameter can be the IP address of the ovirtmgmt interface on the host, and should be reachable by OVN hosts and central server.